server/src/app.ts

```diff
@@ -55,9 +54,11 @@ Sentry.init({
   // Sentry's session tracking keeps the process running, which causes tests to
   // hang. We don't really need session tracking, so turn it off.
   autoSessionTracking: false,
+  ignoreTransactions: ["/health", "/debugme"],
   integrations: [
     new Sentry.Integrations.Http({ tracing: true }),
-    new Tracing.Integrations.Express({ app }),
+    new Sentry.Integrations.Express({ app }),
+    ...Sentry.autoDiscoverNodePerformanceMonitoringIntegrations(),
```

Comment:
Worth noting: this is recommended in the migration guide and shows up in *some* of the Node/Express docs, but not all. In practice, it doesn’t appear to add anything that doesn’t already get added if we don’t call it, and it surprisingly does *not* add the two tracing integrations I’ve manually specified above for HTTP and Express, despite the fact that this function comes from the tracing module and is named to be about tracing (“performance monitoring”).

Specifically, the *only* thing it does is adds Postgres support, but something else in the bowels of Sentry already does that automatically even if we don’t call this. It seems pointless, but maybe it’s doing something that will stop happening automatically later? 🤷